creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -1022,11 +1086,14 @@ class SnapbackSM {
       // Retrieve clock statuses for all users and their current replica sets
       let replicaSetNodesToUserClockStatusesMap
       try {
-        replicaSetNodesToUserClockStatusesMap =
-          await this.retrieveClockStatusesForUsersAcrossReplicaSet(
-            replicaSetNodesToUserWalletsMap,
-            unhealthyPeers
-          )
+        const {
+          replicasToUserClockStatusMap,
+          unhealthyPeers: clockUnhealthyPeers
+        } = await this.retrieveClockStatusesForUsersAcrossReplicaSet(
+          replicaSetNodesToUserWalletsMap
+        )
+        replicaSetNodesToUserClockStatusesMap = replicasToUserClockStatusMap
+        unhealthyPeers = new Set([...unhealthyPeers, ...clockUnhealthyPeers])
```

Comment:
i see what you're doing but had to read through the code inside `retrieveClockStatusesForUsersAcrossReplicaSet()` to follow it, can you document what you're doing + what the `unhealthyPeers` returned from function means ie it is an augmented version of first `unhealthyPeers`

Reply:
also could we improve name from `clockUnhealthyPeers` to smth like `unhealthyPeersFromClockStatusRetrieval`

Reply:
thanks for the suggestion! will do the renaming and push a followup